Add right-to-left and top-to-bottom tests for text-wrap: balance

aikocny opened the pull request:
Inspired from text-wrap-balance-001.html, I changed the array elements to be Arabic (right-to-left) and Chinese (top-to-bottom, right-to-left) characters to support testing for non-English characters.

Commit:
Add right-to-left and top-to-bottom tests for text-wrap: balance

Inspired from text-wrap-balance-001.html, I changed the array elements to be Arabic (right-to-left) and Chinese (top-to-bottom, right-to-left) characters to support testing for non-English characters.

Commit:
Adding title of the test

nt1m commented:
Harness status seems to be error for both of the tests in all browsers (see the checks tab)

Commit:
Fix duplicate subtest name issue

Removed the test name to avoid duplicate subtest name, harness tests should pass now.

aikocny commented:
Fixed it, there was an error with duplicate subtest name.

nt1m commented on css/css-text/white-space/text-wrap-balance-top-to-bottom.html:
```diff
@@ -0,0 +1,35 @@
+  container.appendChild(normal);
+  container.appendChild(balance);
+  test(() => {
+    assert_equals(normal.offsetHeight, balance.offsetHeight);
```
I think you want to compare the `offsetWidth` here given in vertical writing mode height and width are reversed.

aikocny replied:
Good catch, just committed a change.

nt1m commented on css/css-text/white-space/text-wrap-balance-top-to-bottom.html:
```diff
@@ -0,0 +1,35 @@
+<script>
+const container = document.getElementById('container');
+for (const text of [
+    '平衡应该',
```
This seems to fit in a single line within height: 15ch, so balancing won't have an effect on this.

aikocny replied:
Yes it is the intention to test strings that displays as one, two, and three lines. To make sure that balancing will not add the number of lines.

Commit:
Change offsetHeight to offsetWidth

It's in vertical writing mode so we want to compare between the width of the text.
